### PR TITLE
Psswd protect

### PR DESCRIPTION
**Pull request by maiana-sanjuan**
Just added a couple lines of code on the server and ui for password protection when the app is deployed.

### CODE REVIEW

File: admin/create_credentials.R [hunk 1]
```diff
@@ -0,0 +1,9 @@
+# Code to create the credential files
```
Comment (jvillacampa) [inline admin/create_credentials.R:1]: in the projects I use this. I don't commit this file as you could end commiting the login details. I would add it to the gitignore and then share the file with whoever needs to deploy the app. As you have already commited, two options, delete this branch and recreate somewhere else, or delete this file, commit, then add to gitignore and add back to the project. Hope that makes sense

Comment (maiana-sanjuan) [reply admin/create_credentials.R:1]: Thanks @jvillacampa, I'll try deleting the file, commit, and add it to the gitignore

**Review by jvillacampa — COMMENTED** [review]
I'll add here what I sent you through teams:
Hi Maiana, I had a quick look and is not working for me. If I delete the shinyUI function in the ui then it works (not sure why you need this), but then an error appears,

 Input `..1` is `case_when(...)`.
x `TRUE ~ audit == audit` must be length 0 or one, not 3444.
  [No stack trace available]
Maybe you need to look at this to solve it using req() https://stackoverflow.com/questions/67814726/shiny-observe-function-doesnt-work-with-shinymanager



Also, I noticed three other things:

- You load a shapefile using the shapefile packages and all that. A better/quicker way is shaving the shapefile as an rds and then load it as such. It will still work the same way, but you can reduce the number of packages you load and the shapefile will load much quicker.
- You have committed the credentials.rds file, which will contain the password. I would delete this and add it to the gitignore.
- You have got all the data files you need in the repo. Are these going to change often? If so, this could be a problem as every version of the files will make a gigantic repo and is generally not encouraged as well for security reasons. There are sometimes when I have uploaded data files that won't change almost ever, but I keep the core data elsewhere and have scripts that do the data preparation bringing the files the app need to my local repo. Also, you have the repo as public so the data that you are keeping password-protected would be accessible to anyone through here.

Hope that helps

**Review by rmccreath — APPROVED** [review]
Yep, this is all working as I think is expected so happy to approve. 

A couple of points to consider though:
* What is the plan for maintaining users?
* Font Awesome is throwing some warning messages, these are handled but I believe the icons being used aren't representative of their action due to this.

**Pull request merged**